packages/ckeditor5-dev-release-tools/tests/tasks/cleanuppackages.js

```diff
@@ -0,0 +1,686 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const expect = require( 'chai' ).expect;
+const sinon = require( 'sinon' );
+const mockery = require( 'mockery' );
+const fs = require( 'fs-extra' );
+const upath = require( 'upath' );
+const { globSync } = require( 'glob' );
+
+const mockFs = require( 'mock-fs' );
+
+describe( 'dev-release-tools/tasks', () => {
+	describe( 'cleanUpPackages()', () => {
+		let cleanUpPackages, sandbox, stubs;
+
+		beforeEach( () => {
+			sandbox = sinon.createSandbox();
+
+			// Calls to `fs` and `glob` are stubbed, but they are passed through to the real implementation, because we want to test the
+			// real behavior of the script. The whole filesystem is mocked by the `mock-fs` util for testing purposes. A virtual project is
+			// prepared in tests on this mocked filesystem.
+			stubs = {
+				fs: {
+					readJsonSync: sandbox.stub().callsFake( ( ...args ) => fs.readJsonSync( ...args ) ),
```

Comment:
This will not work, because `sinon` does not know what is stubbed (no arguments in `stub()` method) and it says: `Unable to find wrapped method`.

Also I tried with `const readJsonSyncStub = sandbox.stub( fs, 'readJsonSync' ).callThrough()` but then `readJsonSyncStub.callCount` is always 0. It seems that the imported `fs` lib in tests is another instance than the one from the source code.

The only solution that works for me is to use `mockery` and explicitly calling the real methods from the stubs.